core/src/main/java/org/radargun/stages/test/legacy/LegacyTestStage.java

```diff
@@ -96,10 +101,11 @@ public DistStageAck executeOnSlave() {
       try {
          long startNanos = TimeService.nanoTime();
          log.info("Starting test " + testName);
-         List<LegacyStressor> stressors = execute();
+         stressorsManager = setUpAndStartStressors();
+         waitForStressorsToFinish(stressorsManager);
```

Comment:
Couldn't these use the stressorsManager field directly? So you'd call just
setUpAndStartStressors();
waitForStressorsToFinish();